RELEASE/scripts/autoscend/quests/level_08.ash

```diff
@@ -612,7 +612,7 @@ boolean L8_trapperGroar()
 	if(provideResistances(resGoal, false) || provideResistances(resGoal, true))
 	{
 		auto_log_info("Time to take out Gargle, sure, Gargle (Groar)", "blue");
-		addToMaximize("2000cold resistance 5max");
+		equipMaximizedGear();
```

Comment:
doesn't this happen automatically as part of the next line of `return autoAdv`

Reply:
https://kolmafia.us/threads/betweenbattlescript-and-afteradventurescript-can-fail-to-be-run.24764/